Source/WebKit/WebProcess/WebProcess.messages.in

```diff
@@ -229,4 +229,8 @@ messages -> WebProcess : AuxiliaryProcess WantsAsyncDispatchMessage {
     PostNotification(String message, std::optional<uint64_t> state)
     PostObserverNotification(String message)
 #endif
+
+#if ENABLE(CONTENT_EXTENSIONS)
+    SetResourceMonitorContentRuleList(WebKit::WebCompiledContentRuleListData ruleListData)
```

Comment:
Please add `[DeferSendingIfSuspended]` to this IPC. If the receiver is suspended for a long time (like multiple days), we don't want multiple copies of this message building up in the receiver's message queue.